python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -144,3 +146,93 @@ def run(
                 metricName, np.mean(np.abs(result["corr"])), extras=extras
             )
         )
+
+
+class wPerpTableConfig(Config):
+    """Class to organize the yaml configuration parameters to be passed to
+    TExTableTask when using a parquet table input. All values needed to perform
+    TExTableTask have default values set below.
+
+    Optional Input (yaml file)
+    ----------
+    Column names specified as str in yaml configuration for TeX task. These are
+    the desired column names to be passed to the calcuation. If you wish to use
+    values other than the default values specified below, add the following e.g.
+    line to the yaml file:
+
+    config.measure.raColumn = "coord_ra_new"
+    """
+
+    bright_rmag_cut  = Field(
+        doc="Bright magnitude limit to select", dtype=float, default=17.0
+    )
+    faint_rmag_cut = Field(
+        doc="Faint magnitude limit to select", dtype=float, default=23.0
+    )
```

Comment:
Would these be better as SN cuts than magnitude cuts? If you used actions then the user could decide.

Reply:
Again, these mag cuts aren't in the PR'ed version of this ticket branch.

Reply:
Again, these mag cuts aren't on the PR'ed version of the ticket branch.